### ISSUE TITLE
Semi-proportional variant, like iA Writer Duo

### ISSUE THREAD

**Issue by be5invis** [JetBrains/JetBrainsMono#70]
This is really nice for writing documentations:

![image](https://user-images.githubusercontent.com/240091/72765579-ee2e3300-3ba1-11ea-9ff6-6d35e7e1a092.png)

Repository: https://github.com/iaolo/iA-Fonts/tree/master/iA%20Writer%20Duospace
Blog post: https://ia.net/topics/in-search-of-the-perfect-writing-font

**Comment by philippnurullin** [comment]
@be5invis Yes, we were thinking to expand in this way too. Which you prefer the Duo or Quattro? We were leaning toward Quattro approach. It will work best with JetBrains Mono letter construction.

**Comment by auduchinok** [comment]
Please, if implemented, make it a separate font version.
It won't work well in some code formatting scenarios and in languages where vertical alignment is required by syntax (i.e. where you need an actually monospace font).

**Comment by philippnurullin** [comment]
@auduchinok You mean different font families? Yes, definitely.

**Comment by jendrikw** [comment]
For Reference, Latin Modern also has a proportional typewriter font.

https://tug.org/FontCatalogue/latinmodernmonoproportional/
https://www.fontsquirrel.com/fonts/Latin-Modern-Mono

**Comment by Omnieboer** [comment]
Oh, Quattro does look good, I'd say good enough for non-programming implementations anywhere where other sans fonts would work. Looking forward to using Quattro (or Duo if Quattro doesn't exists(yet)) on any web pages and apps I write.

**Comment by arzg** [comment]
I am a user of Input Sans, a proportional font made for code. Take a look at the ‘Proportional fonts? For code?’ section on [Input’s info page](https://input.fontbureau.com/info/). This is much closer to iA Writer Quattro than Duo, as Input Sans doesn’t limit itself to even four different character widths; instead, it just uses whatever is most natural to the letter.

After getting used to Input Sans, I find monospaced fonts distracting as the contortions they make to make all characters the same width are painfully obvious. I’d love to see what a proportional version of JetBrains Mono (maybe JetBrains Sans?) could look like, but I understand it’s a massive undertaking.

**Comment by CodeCrazy-ywt** [comment]
> @auduchinok You mean different font families? Yes, definitely.

Sorry to ask, is it still in progress?

**Comment by pinpox** [comment]
> > @auduchinok You mean different font families? Yes, definitely.
> 
> Sorry to ask, is it still in progress?

@philippnurullin is a proportional variant still planned? Looking for something to combine with JetBrains Mono

**Comment by FlorianFranzen** [comment]
@philippnurullin Any update on this?

**Comment by mattiasdh** [comment]
In my opinion the Trispace font nailed the ideal concept: monospaced, with a slightly wider form (m, w, etc.) and a slightly narrower form (i, l, space, punctuation). Simple but so effectively readable!
Check the backstory and github link @ https://etceteratype.co/trispace

Jetbrains would rock in a Tri variant!
